groovy/groovy.editor/src/org/netbeans/modules/groovy/editor/compiler/error/GroovyError.java

```diff
@@ -117,4 +117,71 @@ public boolean isLineError() {
     public boolean showExplorerBadge() {
         return false;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (obj.getClass() != GroovyError.class) {
+            return false;
+        }
+
+        final GroovyError test = (GroovyError)obj;
+
+        if (this.start != test.start) {
+            return false;
+        }
+
+        if (this.end != test.end) {
+            return false;
+        }
+        
+        if ((this.description == null && test.description != null) || 
+                (this.description != null && test.description == null)
+                || !this.description.equals(test.description)) {
```

Comment:
@ppisl why was this merged without taking reviewer comments into account? If someone takes their time and reviews a PR it would be also good to leave at least a comment why the decision was made to merge it as is.